AdminServer/appscale/admin/__init__.py

```diff
@@ -266,6 +241,25 @@ def version_from_payload(self):
       raise CustomHTTPError(HTTPCodes.BAD_REQUEST,
                             message='Invalid version ID')
 
+    if 'basicScaling' in version or 'manualScaling' in version:
```

Comment:
Yes, manual scaling allows you to keep state in your AppServer, but it prevents you from using the autoscaler.

https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps.services.versions#ManualScaling